docs/api/ethereum.md

```diff
@@ -2456,6 +2456,51 @@ const mutation = await ledger.revokeAsset(assetId);
 
 [destroyAsset](#destroyasset-assetid)
 
+### setAbilities(accountId, abilities)
+
+An `asynchronous` class instance `function` which sets `abilities` of an `accountId`.
+
+::: warning
```

Comment:
::: Warning

Reply:
This is code for yellow warning label it has to be lowercase.